developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Network Identity For FullNode"
+slug: "network-identity-fullnode"
+sidebar_position: 12
+---
+
+# Network Identity For FullNode
+
+This guide will show you how to:
+- Create a static network identity for your FullNode.
+- Retrieve the public network identity.
+- Start a node with (or without) a static network identity.
+
+## Before you proceed
+
+Before you proceed, make sure that you already know how to start your local FullNode. See [Run a FullNode](run-a-fullnode) for detailed documentation.
+
+:::caution Docker support only on Linux
+
+Docker container is currently supported only on Linux x86-64 platform. If you are on macOS or Windows platform, use the Aptos-core source approach.
+
+:::
+
+## Creating a static identity for a FullNode
+
+FullNodes will automatically start up with a randomly generated network identity (a `PeerId` and a public key pair). This works well for regular FullNodes, but you may wish to be added to another node's allowlist, provide specific permissions or run your FullNode with the same identity. In this case, creating a static network identity can help.
+
+1. First, build the `aptos-operational-tool` using the [aptos-labs/aptos-core](https://github.com/aptos-labs/aptos-core) repo.
+
+    - Fork and clone the [aptos-labs/aptos-core](https://github.com/aptos-labs/aptos-core) repo. For example:
+
+    ```
+    $ git clone https://github.com/<YOUR-GITHUB-USERID>/aptos-core.git
+    $ cd aptos-core
+    $ ./scripts/dev_setup.sh
+    $ source ~/.cargo/env
+    ```
+
+    **Using Docker**
+
+    Alternatively, if you are on either macOS or Windows platform, you can use Aptos Docker image. `cd` into the directory for your local public FullNode and start a Docker container with the latest tools, for example:
```

Comment:
Silly question: the caution above suggests that docker is not supported for macOS or Windows, right?

Reply:
Heh. Spoonerism on my part. Thanks for catching this.